tests/test_unit/test_utils.py

```diff
@@ -0,0 +1,51 @@
+import glob
+import tempfile
+
+import pytest
+import yaml
+
+from wazp.utils import df_from_metadata_yaml_files
+
+
+def get_sample_project_metadata_fields() -> dict:
+    """Get the metadata dictionary from the sample project for testing."""
+    with open("sample_project/metadata_fields.yaml") as fi:
+        metadata_fields = yaml.safe_load(fi)
+    return metadata_fields
+
+
+def test_df_from_metadata_yaml_sample_project_metadata() -> None:
+    """Normal operation: test we can read the sample project metadata."""
+    fields = get_sample_project_metadata_fields()
+    testme = df_from_metadata_yaml_files("sample_project/videos", fields)
+
+    expected = set(fields)
+    actual = set(testme.columns)
```

Comment:
I'd suggest more descriptive names for the variables, but this might be personal:
fields --> fields_dict, or metadata_yaml?
testme ---> df_output
expected --> fields_from_yaml
actual ---> df_columns